internal/os/filesystem/api.go

```diff
@@ -31,6 +33,21 @@ func (APIImplementor) PathExists(path string) (bool, error) {
 	return pathExists(path)
 }
 
+func pathValid(path string) (bool, error) {
+	cmd := exec.Command("powershell", "/c", `Test-Path $Env:remoteapth`)
+	cmd.Env = append(os.Environ(), fmt.Sprintf("remoteapth=%s", path))
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return false, fmt.Errorf("returned output: %s, error: %v", string(output), err)
+	}
+
+	return strings.HasPrefix(strings.ToLower(string(output)), "true"), nil
+}
+
+func (APIImplementor) PathValid(path string) (bool, error) {
```

Comment:
pls add some comments here.
I think it worth mentioning the password change use case?

Reply:
done